src/blocks/template-everything/edit.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * BLOCK: Template Everything.
+ */
+import styling from './styling';
+import React, { useEffect, useCallback, useLayoutEffect } from 'react';
+import addBlockEditorDynamicStyles from '@Controls/addBlockEditorDynamicStyles';
+import { useDeviceType } from '@Controls/getPreviewType';
+import { __experimentalBlockVariationPicker } from '@wordpress/block-editor';
+import { createBlock } from '@wordpress/blocks';
+import { useSelect, useDispatch } from '@wordpress/data';
+import Settings from './settings';
+import Render from './render';
+import responsiveConditionPreview from '@Controls/responsiveConditionPreview';
+import { withNotices } from '@wordpress/components';
+import { compose, createHigherOrderComponent } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import './style.scss';
+import styles from './editor.lazy.scss';
+
+const UAGBTemplateEverything = ( props ) => {
+	const deviceType = useDeviceType();
+
+	const {
+		attributes: { UAGHideDesktop, UAGHideTab, UAGHideMob },
+		isSelected,
+		setAttributes,
+		clientId,
+	} = props;
+
+	useEffect( () => {
+		// Assigning block_id in the attribute.
+		setAttributes( { block_id: clientId.substr( 0, 8 ) } );
+	}, [] );
+
+	// Add and remove the CSS on the drop and remove of the component.
+	useLayoutEffect( () => {
+		styles.use();
+		return () => {
+			styles.unuse();
+		};
+	}, [] );
+
+	useEffect( () => {
+		const blockStyling = styling( props );
+
+		addBlockEditorDynamicStyles(
+			'uagb-template-everything-style-' + clientId.substr( 0, 8 ),
+			blockStyling
+		);
+	}, [ props ] );
```

Comment:
```suggestion
	}, [ props.attributes ] );
```

or else if we extract attributes above we can replace props.attributes with attributes everwhere